app/_layouts/oas/index.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+---
+
+<div class="page page-hub oas-landing">
+  <div class="page-hub--header-background"></div>
+  <div class="page-hub--header">
+    <h1>Welcome to Kong API Specs</h1>
+    <h3>This section of our docs site is Powered by Konnect, using <a href="/konnect/dev-portal/customization/">customizations available for the DevPortal</a>.</h3>
```

Comment:
I like that we reference how this page is generated so others can do the same in their company. I'm wondering if we should also call out that this documentation is for Konnect as well as Kong Gateway ("Powered by Konnect" might lead people to think that these are only Konnect APIs?)